woocommerce/woocommerce
# Renamed Local Pickup options on mock data.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

We have two local pickup options in the mock data for the Cart and Checkout preview. This adds confusion as the name is the same, appearing to be a duplicated Local Pickup entry. This PR renames both options.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. On a new instance configure one shipping zone with one shipping method (eg. free shipping)
2. Open the Cart/Checkout page in the editor
3. Observe mocked shipping options displaying `Local Pickup #1` and `Local Pickup #2`


### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Tweak: removed the Los Angeles Local Pickup mock option in the editor preview for Cart and Checkout.
</details>

<details>

<summary>Changelog Entry Comment</summary>

</details>
